test/test.js

```diff
@@ -810,6 +810,17 @@ describe('Page', function() {
       const response = await page.goto(EMPTY_PAGE);
       expect(response.ok).toBe(true);
     }));
+    it('should block patterns', SX(async function() {
+      await page.setRequestInterceptionEnabled(true, ['*.css']);
+      page.on('request', request => {
+        request.continue();
```

Comment:
Shouldn't this be request.abort()?

Reply:
You're right. Fixed in 72800b7.

Reply:
@JoelEinbinder Also, the test would be more rounded if I have more XHRs in a page and I block specific patterns, while leaving out the rest. Should I create another HTML fixture?